eslint.config.js

```diff
@@ -0,0 +1,5 @@
+export default [
+  {
```

Comment:
we already have eslint.config.mjs. Use that file instead.

Reply:
when I do this in mjs file it doesn't work